vlm/src/main/scala/PaddedTile.scala

```diff
@@ -0,0 +1,12 @@
+package geotrellis.contrib.vlm
+
+
+import geotrellis.raster.GridBounds
+
+
+case class PaddedTile(
```

Comment:
this should `extent Tile` so it can be returned as part of `MultibandTile` without changing any of the `RasterSource` signatures.